plugin/core/windows.py

```diff
@@ -339,10 +339,10 @@ def start_async(self, config: ClientConfig, initiating_view: sublime.View) -> No
             self._new_session = session
         except Exception as e:
             message = "".join((
-                "Failed to start subprocess for {0}. Reason:\n\n",
-                "{1}\n\n",
-                "{0} will be disabled for this project. Enable {0} again by running ",
-                "\"LSP: Enable Language Server In Project\" from the Command Palette."
+                "Failed to start subprocess for {0}. {0} will be disabled for this window. ",
+                "Enable {0} again by running \"LSP: Enable Language Server In Project\" from the Command Palette.",
+                "\n\n",
+                "{1}"
```

Comment:
Let's try to make it clear that what follows is the error returned from the server. Maybe just a `Returned error: ` prefix would be enough.